ci-operator/step-registry/openshift-qe/cloud-governance/tagging/ecoeng-01/openshift-qe-cloud-governance-tagging-ecoeng-01-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Export ES environment variables
+es_host="$(cat /secret/es_host)"
+export es_host
+es_port="$(cat /secret/es_port)"
+export es_port
+perf_services_url="$(cat /secret/perf_services_url)"
+export perf_services_url
+
+
+function run_policy(){
+    regions=("us-east-1" "us-east-2" "us-west-1" "us-west-2" "ap-south-1" "eu-north-1" "eu-west-3" "eu-west-2"
+             "eu-west-1" "ap-northeast-3" "ap-northeast-2" "ap-northeast-1" "ca-central-1" "sa-east-1"
+             "ap-southeast-1" "ap-southeast-2" "eu-central-1")
+    for region in "${regions[@]}"; do
+        AWS_DEFAULT_REGION="${region}"
+        export AWS_DEFAULT_REGION
+        export s3_bucket
+        policy_output="s3://${s3_bucket}/${LOGS}/${region}"
+        export policy_output
+        python3 /usr/local/cloud_governance/main.py
+    done
+}
+
+function run_accounts(){
+  account_names=("industry-partners" "certification-pipeline" "ecoengverticals-qe")
```

Comment:
Have `account_names` as a variable or bake them into the vault.
Doing so, we can have common logic, referenced via same chain/step, and keeping the code growth minimum.